pipeline_games/load_games.py

```diff
@@ -0,0 +1,289 @@
+"""Script for loading to database"""
+from os import environ
+from dotenv import load_dotenv
+import pandas as pd
+from psycopg2 import connect, DatabaseError, sql
+from psycopg2.extras import RealDictCursor, execute_batch
+
+
+def get_db_connection(config) -> connect:
+    """Connect to the database with game data"""
+    try:
+        return connect(
+            user=config['DATABASE_USERNAME'],
+            password=config['DATABASE_PASSWORD'],
+            host=config['DATABASE_IP'],
+            port=config['DATABASE_PORT'],
+            database=config['DATABASE_NAME'])
+    except ValueError:
+        return "Error connecting to database."
+
+
+def execute_batch_columns(conn, data: pd.DataFrame, table: str, column: str, page_size=100) -> None:
+    """batch execution of adding specified data to the database"""
+    tuples = list(zip(data.unique()))
+    cols = column
+    query = """INSERT INTO %s(%s) VALUES(%%s)""" % (table, cols)
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        try:
+            execute_batch(cur, query, tuples, page_size)
+            conn.commit()
+            cur.close()
+            print("execute_batch() done")
+        except (Exception, DatabaseError) as error:
+            print(f"Error: {error}")
+            conn.rollback()
+            cur.close()
+
+
+def execute_batch_columns_for_genres(conn, data: pd.DataFrame, table: str, page_size=100) -> None:
+    """batch execution of adding genres into the database"""
+    tuples = [tuple(x) for x in data.to_numpy()]
+    cols = ','.join(list(data.columns))
+    query = "INSERT INTO %s(%s) VALUES(%%s,%%s) " % (table, cols)
+    with conn.cursor(cursor_factory=RealDictCursor) as cur:
+        try:
+            execute_batch(cur, query, tuples, page_size)
+            conn.commit()
+            cur.close()
+            print("execute_batch() done")
+        except (Exception, DatabaseError) as error:
```

Comment:
Same issue here with Exception as well as a specified error

Reply:
Also, if you do decide to replace Exception here with Error earlier mentioned from psycopg2 - I found something kind of cool:

`DatabaseError is a general Python exception class provided by the Python Database API Specification (PEP 249) and is used by database drivers to represent errors related to the database. While psycopg2.Error is a base class for exceptions specific to psycopg2, it does inherit from the DatabaseError class. So, when you catch psycopg2.Error, you are also catching exceptions of type DatabaseError since it's part of the inheritance hierarchy.`